code/game/objects/items/ego_weapons/aleph.dm

```diff
@@ -341,21 +341,21 @@
 			Looking at the E.G.O for more than 3 seconds will make you sick."
 	special = "Using it in hand will activate its special ability. To perform this attack - click on a distant target."
 	icon_state = "censored"
-	force = 75
+	force = 85
 	damtype = BLACK_DAMAGE
 	armortype = BLACK_DAMAGE
 	attack_verb_continuous = list("attacks")
 	attack_verb_simple = list("attack")
 	hitsound = 'sound/weapons/ego/censored1.ogg'
 	attribute_requirements = list(
-							FORTITUDE_ATTRIBUTE = 80,
-							PRUDENCE_ATTRIBUTE = 100,
-							TEMPERANCE_ATTRIBUTE = 80,
-							JUSTICE_ATTRIBUTE = 80
+							FORTITUDE_ATTRIBUTE = 110,
+							PRUDENCE_ATTRIBUTE = 90,
+							TEMPERANCE_ATTRIBUTE = 90,
+							JUSTICE_ATTRIBUTE = 90
```

Comment:
Why the increased requirements? This is essentially on par with most of our Aleph weapons at the moment now, is it not?

Reply:
This is a strange attribute requirement.